app/controllers/work_packages/moves_controller.rb

```diff
@@ -81,32 +59,18 @@ def create
     end
       end
 
-  def set_flash_from_bulk_work_package_save(work_packages, unsaved_work_package_ids)
-    if unsaved_work_package_ids.empty? and not work_packages.empty?
+  def set_flash_from_bulk_work_package_save(work_packages, unsaved_work_packages)
```

Comment:
Assignment Branch Condition size for set_flash_from_bulk_work_package_save is too high. [16.28/15]<br>Line is too long. [81/80]